docs/commands/join.md

```diff
@@ -40,19 +41,19 @@ redocly join --version
 
 ## Options
 
-Option | Type | Description
--- | -- | --
-apis | array | **REQUIRED.**  1. Array of paths to API definition files that you want to join. At least two input files are required.<br />2. A wildcard pattern to match API definition files within a specific folder.
---help | boolean | Show help.
---lint | boolean | Lint definition files.
---decorate | boolean | Run decorators.
---preprocess | boolean | Run preprocessors.
---prefix-tags-with-filename | string | Prefix tags with property value from file name. See the [prefix-tags-with-filename section](#prefix-tags-with-filename) below.
---prefix-components-with-info-prop | string | Prefix components with property value from info object. See the [prefix-components-with-info-prop section](#prefix-components-with-info-prop) below.
---prefix-tags-with-info-prop | boolean | Prefix tags with property value from info object. See the [prefix-tags-with-info-prop](#prefix-tags-with-info-prop) section.
---without-x-tag-groups | boolean | Skip automated `x-tagGroups` creation. See the [without-x-tag-groups](#without-x-tag-groups) section.
---version | boolean | Show version number.
---output, -o | string | Name for the joined output file. Defaults to `openapi.yaml`. **If the file already exists, it's overwritten.**
+| Option                             | Type     | Description                                                                                                                                                                                              |
+| ---------------------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| apis                               | [string] | **REQUIRED.** 1. Array of paths to API definition files that you want to join. At least two input files are required.<br />2. A wildcard pattern to match API definition files within a specific folder. |
```

Comment:
This one is a bit different to the others, because we do allow multiple positional arguments for this specific command. Is it accurate to call it an array? Or should we instead mention that this argument should repeat as many times as needed?

Reply:
I don't think we allow. The only command that accepts multiple positionals is `push`.

Reply:
I think the "array" here is the `api` argument multiple times. The `push` command should be getting an update to remove the multiple positionals (at least officially, even if we still support the old syntax)

Reply:
I'd suggest updating the docs regarding `push` command in scope of this ticket: [Make push command clearer in usage and help text#1032](https://github.com/Redocly/redocly-cli/issues/1032)

Reply:
@tatomyr the `bundle` command above indicates it accepts multiple positional arguments too (and I'm aware people do use that).

Reply:
@adamaltman Yes, it does. Are we discussing the `bundle` or `join` or `push` command here? I've got lost 😄